experimental/saiexperimentaldashacl.h

```diff
@@ -105,46 +106,81 @@ typedef enum _sai_dash_acl_rule_attr_t
      * @type sai_object_id_t
      * @flags MANDATORY_ON_CREATE | CREATE_ONLY
      * @objects SAI_OBJECT_TYPE_DASH_ACL_GROUP
+     * @isresourcetype true
      */
     SAI_DASH_ACL_RULE_ATTR_DASH_ACL_GROUP_ID,
 
     /**
-     * @brief List matched key dip
+     * @brief Ternary matched key dst_tag
      *
-     * @type sai_ip_prefix_list_t
+     * @type sai_uint32_t
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     */
+    SAI_DASH_ACL_RULE_ATTR_DST_TAG,
```

Comment:
ACL tags should be removed from the SAI API. Recently DASH WG decided to remove ACL tags from the SAI requirements

Reply:
i don't follow exactly what rules are for acl, i just generated headers from DASH repo and just added here